app/src/main/java/com/duckduckgo/app/browser/autofill/AutofillCredentialsSelectionResultHandler.kt

```diff
@@ -152,7 +153,9 @@ class AutofillCredentialsSelectionResultHandler @Inject constructor(
     ): LoginCredentials? {
         val selectedCredentials = result.getParcelable<LoginCredentials>(CredentialUpdateExistingCredentialsDialog.KEY_CREDENTIALS) ?: return null
         val originalUrl = result.getString(CredentialUpdateExistingCredentialsDialog.KEY_URL) ?: return null
-        return credentialSaver.updateCredentials(originalUrl, selectedCredentials)
+        val updateType =
```

Comment:
updateType can be null here it seems but AutofillCredentialSaver.updateCredentials() doesn't accept a null CredentialUpdateType. (https://github.com/duckduckgo/Android/pull/2434/files#diff-5464b8c097bf98dc5e4bd9bd3108916a2a9800e28630d2b1d1a38480ed2be433R185)

Reply:
I guess same goes for originalUrl and selectedCredentials.